src/patternfly/upgrade-examples/LoginUpgradeExamples/docs/code.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+
+When converting PatternFly 3 components to PatternFly 4 components, you must also take into consideration the layouts and sizings that PatternFly 3 utilized from Bootstrap 3.
+
+## Usage
+
+| PF3 Class | Applied To | PF4 Class | Applied To | Outcome |
+| -- | -- | -- | -- | -- |
+| `.login-pf` | `<body>` | `.pf-c-login` | `<div>` | Parent container that holds various login components. |
+| `.container-fluid`| `<div>` | `.pf-c-login__container`| `<div>` | Primary container that holds the login component itself. |
+| `.login-pf-page-header` | `<header>` |`.pf-c-login__header` | `<header>` | Header container that sits outside of the login component. |
+| -- | -- | `.pf-c-login__main` | `<main>` | Main login container. |
+| -- | -- | `.pf-c-login__main-body` | `<div>` | Main form container. |
+| `.login-pf-brand`| `<img>` | `.pf-c-brand` | `<img>` | Patternfly logo brand image tag. |
+| `.login-pf-header` | `<header>` | `.pf-c-login__main-header` | `<header>` | Header container that sits inside of the login component. |
+| `.card-pf` > `.login-pf-signup` | `<p>` | -- | -- | PF3 signup section. |
```

Comment:
should you add `.login-pf-settings`?

Reply:
PF3's `.login-pf-settings` class comparison with PF4 version has been added. The only thing that is miss which within this container is the "Forgot Password" link, which @christiemolloy had mentioned before already. This is something we will need to open a new issue for once @mcarrano can suggest a better design idea? Thoughts?